lib/cloak_ecto/types/pbkdf2.ex

```diff
@@ -1,219 +1,211 @@
-        end
+      defp put_hashed_fields(changeset) do
+        changeset
+        |> put_change(:field_name_hash, get_field(changeset, :field_name))
+      end
 
-    Ensure that the hash is updated whenever the target field changes with the
-    `put_change/3` function:
+  Query the Repo using the `:field_name_hash` in any place you would typically
+  query by `:field_name`.
 
-        def changeset(struct, attrs \\\\ %{}) do
-          struct
-          |> cast(attrs, [:field_name, :field_name_hash])
-          |> put_hashed_fields()
-        end
+      user = Repo.get_by(User, email_hash: "user@email.com")
+  """
 
-        defp put_hashed_fields(changeset) do
-          changeset
-          |> put_change(:field_name_hash, get_field(changeset, :field_name))
-        end
+  @typedoc "Digest algorithms supported by Cloak.Field.PBKDF2"
+  @type algorithms :: :md4 | :md5 | :ripemd160 | :sha | :sha224 | :sha256 | :sha384 | :sha512
 
-    Query the Repo using the `:field_name_hash` in any place you would typically
-    query by `:field_name`.
+  @doc """
+  Configures the `PBKDF2` field using runtime information.
 
-        user = Repo.get_by(User, email_hash: "user@email.com")
-    """
+  ## Example
 
-    @typedoc "Digest algorithms supported by Cloak.Field.PBKDF2"
-    @type algorithms :: :md4 | :md5 | :ripemd160 | :sha | :sha224 | :sha256 | :sha384 | :sha512
+      @impl Cloak.Ecto.PBKDF2
+      def init(config) do
+        config = Keyword.merge(config, [
+          algorithm: :sha256,
+          secret: System.get_env("PBKDF2_SECRET")
+        ])
 
-    @doc """
-    Configures the `PBKDF2` field using runtime information.
+        {:ok, config}
+      end
+  """
+  @callback init(config :: Keyword.t()) :: {:ok, Keyword.t()} | {:error, any}
 
-    ## Example
+  @doc false
+  defmacro __using__(opts) do
+    otp_app = Keyword.fetch!(opts, :otp_app)
 
-        @impl Cloak.Ecto.PBKDF2
-        def init(config) do
-          config = Keyword.merge(config, [
-            algorithm: :sha256,
-            secret: System.get_env("PBKDF2_SECRET")
-          ])
-
-          {:ok, config}
-        end
-    """
-    @callback init(config :: Keyword.t()) :: {:ok, Keyword.t()} | {:error, any}
-
-    @doc false
-    defmacro __using__(opts) do
-      otp_app = Keyword.fetch!(opts, :otp_app)
-
-      quote do
-        @behaviour Cloak.Ecto.PBKDF2
-        @behaviour Ecto.Type
-        @algorithms ~w[
-          md4
-          md5
-          ripemd160
+    quote do
+      @behaviour Cloak.Ecto.PBKDF2
+      @behaviour Ecto.Type
+      @algorithms ~w[
           sha
           sha224
           sha256
           sha384
           sha512
         ]a
 
-        @impl Cloak.Ecto.PBKDF2
-        def init(config) do
-          defaults = [algorithm: :sha256, iterations: 10_000, size: 32]
+      @impl Cloak.Ecto.PBKDF2
+      def init(config) do
+        defaults = [algorithm: :sha256, iterations: 10_000, size: 32]
```

Comment:
OWASP [suggests 600,000](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#pbkdf2) iterations for `SHA256` and 210,000 iterations for `SHA512`. Would you be amenable to bumping the default number of iterations and also updating the docs?

Reply:
I'll do that in a separate PR, so that it will show up in the automatic changelog more easily.